roles/docker_node/tasks/docker-centos.yml

```diff
@@ -0,0 +1,67 @@
+- name: Replace mirrorlist with commented mirrorlist in CentOS-Base.repo
+  replace:
+    path: /etc/yum.repos.d/CentOS-Base.repo
+    regexp: "^mirrorlist="
+    replace: "#mirrorlist="
+  become: true
+
+- name: Uncomment and update baseurl to point to vault.centos.org
+  replace:
+    path: /etc/yum.repos.d/CentOS-Base.repo
+    regexp: "^#baseurl=http://mirror.centos.org"
+    replace: "baseurl=http://vault.centos.org"
+  become: true
+
+- name: Install yum-utils for repository management
+  yum:
+    name: yum-utils
+    state: present
+  become: true
+
+- name: Add Docker CE repository for CentOS
+  command: yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+  args:
+    creates: /etc/yum.repos.d/docker-ce.repo
+  become: true
+
+- name: install docker packages
+  package:
+    name: "{{ item }}"
+    state: present
+  become: true
+  register: docker_install_results
+  loop:
+    - "{{ docker_ce_cli_package }}"
+    - "{{ containerd_io_package }}"
+    - "{{ docker_ce_package }}"
+    - jq
```

Comment:
Nit: Should the jq install be in a prerequesites task similar to the docker-ubuntu role?